packages/samples/atomic-react/src/components/AtomicPageWrapper.tsx

```diff
@@ -56,22 +58,34 @@ export const AtomicPageWrapper: FunctionComponent<Props> = ({
   children,
   options = {},
 }) => {
+  const searchConfig = options.expression
+    ? {}
+    : {
+        pipeline: 'Search',
+        searchHub: 'MainSearch',
+      };
```

Comment:
This will be re-evaluated every time the component re-renders, even though it's generally only going to be used once. I recommend moving this expression inside the `useMemo` below.

Reply:
Makes sense, it would also go together with the existing options.

Reply:
Actually the source containing folded results errors when search config is specified
```
{
  "statusCode": 400,
  "message": "Unknown query pipeline: name or id \"Search\"",
  "type": "UnknownQueryPipelineOtherException",
```

Reply:
Yeah, you need to set the pipeline to default (empty string).

Reply:
Updated props as discussed.